### PR TITLE
Small changes to improve document readabilty

### PR DESCRIPTION
**Pull request by lucaslra**
Some small changes in the project documents to improve readability and to link to some resources directly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ The [`vscode`](https://github.com/microsoft/vscode) repository is where we do de
 ## Contributing
 If you are interested in fixing issues and contributing directly to the code base, please see the document [How to Contribute](https://github.com/Microsoft/vscode/wiki/How-to-Contribute), which covers the following:
 
-* How to build and run from source
-* The development workflow, including debugging and running tests
-* [Coding guidelines](https://github.com/Microsoft/vscode/wiki/Coding-Guidelines)
-* Submitting pull requests
+* [How to build and run from source](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#build-and-run-from-source)
+* [The development workflow, including debugging and running tests](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#development-workflow)
+* [Coding Guidelines](https://github.com/Microsoft/vscode/wiki/Coding-Guidelines)
```
Comment (Tyriar) [inline README.md]: "Guidelines" is capitalized differently to the rest of the list, lower case fits better.

**Comment by mattn** [comment]
test fail

**Comment by mattn** [comment]
sorry this change just change docs. but seems this seems to be trigger of error

https://travis-ci.org/Microsoft/vscode/builds

**Comment by joaomoreno** [comment]
@mattn There's a bit of flakiness with our current builds, they seem to fail occasionally, unrelated to your changes. Will address it **fast**.

**Pull request merged**